var/spack/repos/builtin/packages/py-xtb/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyXtb(PythonPackage):
+    """Python API for the extended tight binding program package"""
+
+    homepage = "https://xtb-python.readthedocs.org"
+    url = "https://github.com/grimme-lab/xtb-python/releases/download/v22.1/xtb-22.1.tar.gz"
+
+    maintainers = [
+        *PythonPackage.maintainers,
+        "awvwgk",
+    ]
+
+    version("22.1", sha256="7a59e7b783fc6e8b7328f55211de681e535a83991b07c4bab73494063f5e9018")
+
+    depends_on("pkgconfig", type="build")
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-cffi", type=("build", "run"))
+    depends_on("py-meson-python@0.11.0:", type="build")
+    depends_on("py-numpy", type="run")
```

Comment:
```suggestion
    depends_on("py-numpy", type=("build", "run"))
```

Reply:
Numpy is actually not needed at build time.

Reply:
It is if you run `spack install --test=root py-xtb`. If anyone can figure out how to add run-time deps to the path during a post-install test let me know, but for now it needs to be both.

Reply:
Hmm, other packaging systems would have something like `type="test"`.

Reply:
Spack also has a `type="test"`: https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-types

We _could_ convert every `type=("build", "run")` to `type=("test", "run")` but it feels kinda silly. We could also drop the ability to test Python imports during installation and only allow them with `spack test` but I don't love that solution either. I wish Spack would just know how to handle this.

Reply:
Also, some build tools will check `PYTHONPATH` at build-time to make sure the dependencies are actually installed. So they do need to be build deps regardless.